RELEASE/scripts/sl_ascend/sl_cooking.ash

```diff
@@ -1648,111 +1784,146 @@ boolean loadConsumables(string _type, item[int] item_backmap, int[int] cafe_back
 				howmany = min(howmany, max(0, creatable_amount(it) - sl_reserveCraftAmount(it)));
 				craftables[it] = howmany;
 			}
+			if (storage_amount(it) > 0)
```

Comment:
We might also want to check the mall, at the very least for key pies, so people don't have to manually stock up on them ahead of run.